seacrowd/sea_datasets/mywsl2023/mywsl2023.py

```diff
@@ -0,0 +1,181 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from typing import Dict, List, Tuple
+
+import datasets
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import TASK_TO_SCHEMA, Licenses, Tasks
+
+_CITATION = """\
+@article{JOHARI2023109338,
+title = {MyWSL: Malaysian words sign language dataset},
+journal = {Data in Brief},
+volume = {49},
+pages = {109338},
+year = {2023},
+issn = {2352-3409},
+doi = {https://doi.org/10.1016/j.dib.2023.109338},
+url = {https://www.sciencedirect.com/science/article/pii/S2352340923004560},
+author = {Rina Tasia Johari and Rizauddin Ramli and Zuliani Zulkoffli and Nizaroyani Saibani},
+keywords = {Dataset, Hand gestures, Sign language, Image data},
+}
+"""
+
+_DATASETNAME = "mywsl2023"
+
+_DESCRIPTION = """\
+This dataset contains pictures of hand gestures corresponding to ten commonly-used Malaysian Sign Language (XML) words.
+Gestures are performed by five university students who belong to different ethnic groups and are proficient in XML.
+Each gesture class contains 350 instances.
+"""
+
+_HOMEPAGE = "https://data.mendeley.com/datasets/zvk55p7ktd/1"
+
+_LANGUAGES = ["xml"]  # We follow ISO639-3 language code (https://iso639-3.sil.org/code_tables/639/data)
+
+_LICENSE = Licenses.CC_BY_4_0.value
+
+_LOCAL = False
+
+_URLS = {_DATASETNAME: "https://data.mendeley.com/public-files/datasets/zvk55p7ktd/files/7f11b8a0-24e4-45df-af3d-e861f41435ea/file_downloaded"}
+
+_SUPPORTED_TASKS = [Tasks.SIGN_LANGUAGE_RECOGNITION]
+_SUPPORTED_SCHEMA_STRINGS = [f"seacrowd_{str(TASK_TO_SCHEMA[task]).lower()}" for task in _SUPPORTED_TASKS]
+
+_SPLITS = [datasets.Split.TRAIN, datasets.Split.TEST]
+_SPLIT_NAMES = {
+    datasets.Split.TRAIN: "train",
+    datasets.Split.TEST: "test",
+}
```

Comment:
no need to use this `_SPLIT_NAME`, uses this in `_split_generators`:
```python
#following line will return "test"
datasets.Split.TEST._name

#following line will return "train"
datasets.Split.TRAIN._name
```

Reply:
@Enliven26 I'm suggesting to just follow the template regarding `_split_generators` logical handling. I know that you probably want to be DRY here, but in this case, it is more about readability IMO.

Reply:
yes I agree on this one